### PR TITLE
Porfolio:,Add work and about section

### PR DESCRIPTION
**Pull request by V0pe**
**Mobile Portfolio: Add work and about section**

In this branch:
- Added the mark up for the work and about me section.
- Added the styles for the sections.

### CODE REVIEW

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# INVALID Code Review Request :stop_sign: 

Hi @V0pe , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

- [x] Please consider setting up the linters correctly. At the moment, your linters are set up for JavaScript as well when you deal only with HTML and CSS. Here is the link for the correct linters: [HTML & CSS linters](https://github.com/microverseinc/linters-config/tree/master/html-css)

- [x] Kindly rename your Pull Request with a more representative name for the work you done in this branch. The name "Portfolio #2" is not suggesting much. A more representative name could be "Add Work and About sections"

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

**Comment by V0pe** [comment]
@nicupop729 Thanks for your observation. I have checked the linters and I set the linters.yml in the github workflow. Also I add the .hintrc and .stylelintrc.json for local linter checking. I have also changed the name of the pull request header

**Comment by V0pe** [comment]
@nicupop729 Kindly checkout the linters check on the github. It ticked green

**Comment by nicupop729** [comment]
@V0pe 
You are doing a great job. 
If everything is set, you can ask for other review!
Good luck!

File: index.html [hunk 1]
```diff
@@ -52,5 +52,149 @@ <h1>Hey there. I'm <br>Victor<br><span>
         </div>
         
     </section>
+    <section class="work">
+        <div class="work-title">
+            <h2>My Recent <br>Works</h2>
+            <img src="images/Indicator.png" alt="indicator line">
```
Comment (BenMukebo) [inline index.html:58]: - [ ] Please take a look at this image right here, it is broken and not displaying normally. kindly fix it.
- [ ] Please center the arrow icon, it is a bit more aligned to the left 
my

File: index.html [hunk 2]
```diff
@@ -52,5 +52,149 @@ <h1>Hey there. I'm <br>Victor<br><span>
...
+                feature or website Look through<br>
+                some of my work and experience! If<br>
+                you like what you see and have a<br>
+                project you need coded,<br> 
+                don’t hestiate to contact me.</p>
+            <button class="resume" type="button">Get My Resume</button>
+        </div>
+        <div class="technologies">
+            <div class="tech languages">
+                <div class="icon"><img src="images/Icons_lang.svg" alt="icon-language"></div>
```
Comment (BenMukebo) [inline index.html:160]: - [ ] Your design is to salute 👍🏻 , but I think there are a few things to improve before you go to the next project,
      - fix your background for this section as per the original design
      - all your language images are broken too, please fix them up.
      - 
aboutmy

- it should be like this ⬇️ 
figmyy

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@V0pe,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!




### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Professional README ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (V0pe) [reply index.html:58]: @BenMukebo I have fixed the position. Thanks.

File: index.html (hunk 2 above)
Comment (V0pe) [reply index.html:160]: Ahn.. Apparently i didnt push the images.  Thanks. Fixed it now.

Comment (V0pe) [reply index.html:160]: @BenMukebo Apparently I didnt push the images to github. But its fixed now. Kindly preview

File: index.html (hunk 1 above)
Comment (karvel-code) [inline index.html:58]: - [ ] Great working on this section as requested by the previous reviewer. However, in smaller screen sizes, the `About Me` is overlapping the paragraph below it. Please fix that. 
- See below:
image

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @V0pe , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- You have a professional README file. 
- You have followed git-flow 💯 .
- All your linter checks pass.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (V0pe) [reply index.html:58]: @karvel-code  Thanks Elvis, I have adjusted the font-size and padding. Can you please quickly have a look before I submit it again

**Review by Mre55 — APPROVED** [review]
**Hi @V0pe,**

# Status: APPROVED 🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**